modules/ROOT/pages/application-deployment.adoc

```diff
@@ -0,0 +1,88 @@
+= Application Deployment
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: deploy, application, applications, runtime, amc, cloudhub, on premises, on premise
+
+== Starting Mule
+
+Start Mule by running `<MULE_HOME>/bin/mule` or starting Mule as a service. By default, applications in the `<MULE_HOME>/apps` directory are deployed. You can also designate specific applications to start (separated by a colon - **:**) and Mule 4 respects the order when starting the applications. In this scenario, only the specified applications start.
+
+Start Mule by specifying an app to run:
+
+----
+mule -M-Dmule.deploy.applications=foo
+----
+
+Where `foo` is a Mule app at `$MULE_HOME/apps/foo`.
+
+From this moment, Mule checks every three seconds for the `$MULE_HOME/apps/foo/mule-config.xml` updates. You can update the application `jar` contents and modify this file so Mule reloads the config and class modifications.
+
+Hot deployment doesn’t work when using the `mule.deploy.applications` property.  Only applications in the apps directory are deployed, new or updated applications don't start until Mule restarts.
```

Comment:
in the apps directory > in the `apps` directory. Please check that file names and directories are enclosed in backticks.